azure-pipelines.yml

```diff
@@ -31,6 +31,7 @@ pr:
 variables:
   CIBW_MANYLINUX_X86_64_IMAGE: manylinux1
   CIBW_MANYLINUX_I686_IMAGE: manylinux1
+  CI: true
```

Comment:
This is also in #11089 , which is milestoned for 4.2.1. FYI.

Reply:
Ah thanks!

Reply:
I've now backported that PR correctly and rebased this PR